src/open_inwoner/utils/glom.py

```diff
@@ -0,0 +1,13 @@
+from glom import glom
+
+
+def glom_multiple(obj, paths: tuple[str, ...], *, default=None):
+    if not len(paths) > 1:
+        raise ValueError("no paths provided")
```

Comment:
A bit misleading, as someone might pass a single path, thinking that `glom_multiple` works for both cases. Perhaps `ValueError("glom_multiple requires a tuple of at least two paths")`.